source/help/messaging/keyboard-shortcuts.rst

```diff
@@ -21,6 +21,8 @@ Navigation
 +----------------------------------------+----------------------------------------+------------------------------------------------------------------------------+
 | CTRL+K                                 | CMD+K                                  | Open a quick channel switcher dialog                                         |
 +----------------------------------------+----------------------------------------+------------------------------------------------------------------------------+
+| CTRL+SHIFT+K                           | CMD+SHIFT+K                            | Open the Direct Messages dialog                                              |
```

Comment:
Actually, should it be capitalized? We don't capitalize `account settings` in the CTRL+SHIFT+A shortcut description